SpatialGDK/Documentation/content/toolbars.md

```diff
@@ -36,6 +36,29 @@ You can switch back by unchecking the boxes.
 
 You can launch multiple servers at the same time from within the Unreal Editor in [PIE (Unreal documentation)](https://docs.unrealengine.com/en-us/Engine/UI/LevelEditor/InEditorTesting#playineditor) configuration. To configure the number of servers launched, open the **Play** drop-down menu and use the slider `Number of Servers` within the `Multiplayer Options` section.
 
+To connect multiple servers-workers to SpatialOS, we need to tell SpatialOS how many server-workers we will be connecting. In `<ProjectPath>\spatial\default_launch.json` there is a load balancing section which dictates how many workers will be connected to SpatialOS. 
```

Comment:
Does this whole new section assume that the user has switched to LBV2? Shouldn't we mention somewhere that there's a new load balancing system that you have to upgrade to, and you have to upgrade to the new bridge first, etc.?

Reply:
`default_launch.json` uses LBV2 by default.

Reply:
Ah right - do we make that clear anywhere in the GDK docs? I think we should - even if it's just a note wherever relevant (which might be multiple places).